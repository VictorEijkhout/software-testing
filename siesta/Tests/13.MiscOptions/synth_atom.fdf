%include ../base_wat_smallbox.fdf
SystemLabel     synth_atom

%block ChemicalSpeciesLabel
 1  201  ON-0.34560
 2    1  H
%endblock ChemicalSpeciesLabel
%block SyntheticAtoms
 1
 2 2 3 4
 2.0 3.3456 0.0 0.0
%endblock SyntheticAtoms

