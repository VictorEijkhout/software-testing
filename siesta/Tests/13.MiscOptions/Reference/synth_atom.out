Siesta Version  : 5.0.0-alpha-123-gfb8fd7d02
Architecture    : ----
Compiler version: GNU-11.4.0
Compiler flags  : -fallow-argument-mismatch;-O3 -march=native
PP flags        : ----
Libraries       : ----
Parallelisations: none
GEMM3M support
NetCDF support
NetCDF-4 support
Lua support

Runtime information:
* Directory : /home/fnpedron/Siesta/siesta-dev/Tests/13.MiscOptions/work/synth_atom
* Running in serial mode.
>> Start of run:  28-SEP-2023  15:40:32

                           *********************** 
                           *  WELCOME TO SIESTA  * 
                           *********************** 

reinit: Reading from standard input
reinit: Dumping input in INPUT_TMP.25944
************************** Dump of input data file ****************************
%include ../../basejob_2.fdf
SystemLabel     synth_atom
%block ChemicalSpeciesLabel
 1  201  ON-0.34560
 2    1  H
%endblock ChemicalSpeciesLabel
%block SyntheticAtoms
 1
 2 2 3 4
 2.0 3.3456 0.0 0.0
%endblock SyntheticAtoms
************************** End of input data file *****************************

reinit: -----------------------------------------------------------------------
reinit: System Name: 2 Water molecules
reinit: -----------------------------------------------------------------------
reinit: System Label: synth_atom
reinit: -----------------------------------------------------------------------

initatom: Reading input for the pseudopotentials and atomic orbitals ----------
Species number:   1 Atomic number:  201 Label: ON-0.34560
Species number:   2 Atomic number:    1 Label: H


---- Processing specs for species: ON-0.34560
Reading pseudopotential information in formatted form from:
  /home/fnpedron/Siesta/siesta-dev/Tests/Pseudos/ON-0.34560.psf

---- Processing specs for species: H
Ground state valence configuration:   1s01
Reading pseudopotential information in formatted form from:
  /home/fnpedron/Siesta/siesta-dev/Tests/Pseudos/H.psf
Ground state valence configuration (synthetic):   2s 2.00000  2p 3.34560
Number of synthetic species:  1
WARNING: Assuming absence of semicore states for ON-0.34560
WARNING: If there are any, use the PAO.Basis block

---- Pseudopotential check for H 

Pseudized shells:
1s( 1.00) rc: 1.25
2p( 0.00) rc: 1.25
3d( 0.00) rc: 1.25
4f( 0.00) rc: 1.25
Valence configuration for ps generation: (assumed as above)
For ON-0.34560, standard SIESTA heuristics set lmxkb to 3
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.
For H, standard SIESTA heuristics set lmxkb to 2
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.

<basis_specs>
===============================================================================
ON-0.34560           Z= 201    Mass= -1.0000        Charge= 0.17977+309
Lmxo=2 Lmxkb= 3    BasisType=split      Semic=F
L=0  Nsemic=0  Cnfigmx=2
          i=1  nzeta=1  polorb=0  (2s)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
L=1  Nsemic=0  Cnfigmx=2
          i=1  nzeta=1  polorb=1  (2p)  (to be polarized perturbatively)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
L=2  Nsemic=0  Cnfigmx=3
          i=1  nzeta=0  polorb=0  (3d)  (perturbative polarization orbital) (from 2p)
-------------------------------------------------------------------------------
L=0  Nkbl=1  erefs: 0.17977+309
L=1  Nkbl=1  erefs: 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
L=3  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

Using PAO.SplitTailNorm
atom: Called for (synthetic) ON-0.34560            (Z = 201)
Maximum radius (at nrval) set to   50.00000 8518

read_vps: Pseudopotential generation method:
read_vps: MIXPS     22-JUL-06  ---      ATM 3.2.2 22-JUL-06 
Valence charge in pseudo generation:    5.34560

xc_check: Exchange-correlation functional:
xc_check: Ceperley-Alder
V l=0 = -2*Zval/r beyond r=  1.1822
V l=1 = -2*Zval/r beyond r=  1.1763
V l=2 = -2*Zval/r beyond r=  1.1775
V l=3 = -2*Zval/r beyond r=  1.1727
All V_l potentials equal beyond r=  1.1332
This should be close to max(r_c) in ps generation
All pots = -2*Zval/r beyond r=  1.1822
Using small-core scheme (local charge) for Vlocal

VLOCAL_FROM_CHLOCAL: 99.0% of the norm of Vloc inside     33.803 Ry
VLOCAL_FROM_CHLOCAL: 99.9% of the norm of Vloc inside     77.038 Ry
atom: Maximum radius for chlocal:    1.36689
atom: Maximum radius for r*vlocal+2*Zval:    1.24839

KBgen: Kleinman-Bylander projectors: 
GHOST: No ghost state for L =  0
   l= 0   rc=  1.366893   el= -1.466572   Ekb=  9.401679   kbcos=  0.308959
GHOST: No ghost state for L =  1
   l= 1   rc=  1.365517   el= -0.575298   Ekb= -7.142655   kbcos= -0.351091
GHOST: No ghost state for L =  2
   l= 2   rc=  1.565905   el=  0.013296   Ekb= -1.871242   kbcos= -0.003974
GHOST: No ghost state for L =  3
   l= 3   rc=  1.685249   el=  0.019582   Ekb= -0.740741   kbcos= -0.000392

KBgen: Total number of Kleinman-Bylander projectors:  16
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 2s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    3.827533
                 energy =   -1.455381
                kinetic =    1.337203
    potential(screened) =   -2.792583
       potential(ionic) =   -9.122468

SPLIT: Orbitals with angular momentum L= 1

SPLIT: Basis orbitals for state 2p

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    4.602776
                 energy =   -0.564248
                kinetic =    4.082073
    potential(screened) =   -4.646321
       potential(ionic) =  -10.818000

POLgen: Perturbative polarization orbital with L=  2

POLgen: Polarization orbital for state 2p
Split based on tail norm

   izeta = 1
                     rc =    4.602776
                 energy =    1.839949
                kinetic =    3.585134
    potential(screened) =   -1.745185
       potential(ionic) =   -6.671574
atom: Total number of Sankey-type orbitals:  9

atm_pop: Valence configuration (for local Pseudopot. screening):
 2s( 2.00)                                                            
 2p( 3.35)                                                            
 3d( 0.00)                                                            
Vna: chval, zval:    5.34560   5.34560

Vna:  Cut-off radius for the neutral-atom potential:   4.602776

atom: _________________________________________________________________________

<basis_specs>
===============================================================================
H                    Z=   1    Mass=  1.0080        Charge= 0.17977+309
Lmxo=1 Lmxkb= 2    BasisType=split      Semic=F
L=0  Nsemic=0  Cnfigmx=1
          i=1  nzeta=1  polorb=1  (1s)  (to be polarized perturbatively)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
L=1  Nsemic=0  Cnfigmx=2
          i=1  nzeta=0  polorb=0  (2p)  (perturbative polarization orbital) (from 1s)
-------------------------------------------------------------------------------
L=0  Nkbl=1  erefs: 0.17977+309
L=1  Nkbl=1  erefs: 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

Using PAO.SplitTailNorm
atom: Called for H                     (Z =   1)
Maximum radius (at nrval) set to   50.00000 8518

read_vps: Pseudopotential generation method:
read_vps: ATM3      Troullier-Martins                       
Valence charge in pseudo generation:    1.00000

xc_check: Exchange-correlation functional:
xc_check: Ceperley-Alder
V l=0 = -2*Zval/r beyond r=  1.2384
V l=1 = -2*Zval/r beyond r=  1.2321
V l=2 = -2*Zval/r beyond r=  1.2272
All V_l potentials equal beyond r=  1.2371
This should be close to max(r_c) in ps generation
All pots = -2*Zval/r beyond r=  1.2384
Using small-core scheme (local charge) for Vlocal

VLOCAL_FROM_CHLOCAL: 99.0% of the norm of Vloc inside     28.363 Ry
VLOCAL_FROM_CHLOCAL: 99.9% of the norm of Vloc inside     64.639 Ry
atom: Maximum radius for chlocal:    1.44620
atom: Maximum radius for r*vlocal+2*Zval:    1.31026

KBgen: Kleinman-Bylander projectors: 
GHOST: No ghost state for L =  0
   l= 0   rc=  1.364142   el= -0.467325   Ekb= -2.009475   kbcos= -0.336680
GHOST: No ghost state for L =  1
   l= 1   rc=  1.430269   el=  0.007562   Ekb= -0.503817   kbcos= -0.022554
GHOST: No ghost state for L =  2
   l= 2   rc=  1.469686   el=  0.013286   Ekb= -0.191752   kbcos= -0.002430

KBgen: Total number of Kleinman-Bylander projectors:   9
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 1s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    5.285354
                 energy =   -0.456120
                kinetic =    0.900445
    potential(screened) =   -1.356565
       potential(ionic) =   -1.890080

POLgen: Perturbative polarization orbital with L=  1

POLgen: Polarization orbital for state 1s
Split based on tail norm

   izeta = 1
                     rc =    5.285354
                 energy =    0.642323
                kinetic =    1.282472
    potential(screened) =   -0.640149
       potential(ionic) =   -1.117275
atom: Total number of Sankey-type orbitals:  4

atm_pop: Valence configuration (for local Pseudopot. screening):
 1s( 1.00)                                                            
 2p( 0.00)                                                            
Vna: chval, zval:    1.00000   1.00000

Vna:  Cut-off radius for the neutral-atom potential:   5.285354

atom: _________________________________________________________________________

prinput: Basis input 

PAO.BasisType split     

%block ChemicalSpeciesLabel
    1  201 ON-0.34560              # Species index, atomic number, species label
    2    1 H                       # Species index, atomic number, species label
%endblock ChemicalSpeciesLabel

%block SyntheticAtoms   # Valence config
  1
  2  2  3  4
  2.00000  3.34560  0.00000  0.00000
%endblock SyntheticAtoms

%block PAO.Basis                 # Define Basis set
# WARNING: This information might be incomplete!
ON-0.34560            2                    # Species label, number of l-shells
 n=2   0   1                         # n, l, Nzeta 
   3.828   
   1.000   
 n=2   1   1 P   1                   # n, l, Nzeta, Polarization, NzetaPol
   4.603   
   1.000   
H                     1                    # Species label, number of l-shells
 n=1   0   1 P   1                   # n, l, Nzeta, Polarization, NzetaPol
   5.285   
   1.000   
%endblock PAO.Basis

prinput: ----------------------------------------------------------------------

Dumping basis to NetCDF file ON-0.34560.ion.nc
Dumping basis to NetCDF file H.ion.nc
coor:   Atomic-coordinates input format  =     Cartesian coordinates
coor:                                          (in Angstroms)

siesta: Atomic coordinates (Bohr) and species
siesta:      0.00000   0.00000   0.00000  1        1
siesta:      1.43052   1.10738   0.00000  2        2
siesta:     -1.43052   1.10738   0.00000  2        3
siesta:      0.00000   9.44863   0.00000  1        4
siesta:      1.43052   8.34125   0.00000  2        5
siesta:     -1.43052   8.34125   0.00000  2        6

siesta: System type = slab      

initatomlists: Number of atoms, orbitals, and projectors:      6    34    68

siesta: ******************** Simulation parameters ****************************
siesta:
siesta: The following are some of the parameters of the simulation.
siesta: A complete list of the parameters used, including default values,
siesta: can be found in file out.fdf
siesta:
redata: Spin configuration                          = none
redata: Number of spin components                   = 1
redata: Time-Reversal Symmetry                      = T
redata: Spin spiral                                 = F
redata: Long output                                 =   F
redata: Number of Atomic Species                    =        2
redata: Charge density info will appear in .RHO file
redata: Write Mulliken Pop.                         = NO
redata: Matel table size (NRTAB)                    =     1024
redata: Mesh Cutoff                                 =   300.0000 Ry
redata: Net charge of the system                    =     0.0000 |e|
redata: Min. number of SCF Iter                     =        0
redata: Max. number of SCF Iter                     =     1000
redata: SCF convergence failure will abort job
redata: SCF mix quantity                            = Hamiltonian
redata: Mix DM or H after convergence               =   F
redata: Recompute H after scf cycle                 =   F
redata: Mix DM in first SCF step                    =   T
redata: Write Pulay info on disk                    =   F
redata: New DM Occupancy tolerance                  = 0.000000000001
redata: No kicks to SCF
redata: DM Mixing Weight for Kicks                  =     0.5000
redata: Require Harris convergence for SCF          =   F
redata: Harris energy tolerance for SCF             =     0.000100 eV
redata: Require DM convergence for SCF              =   T
redata: DM tolerance for SCF                        =     0.000100
redata: Require EDM convergence for SCF             =   F
redata: EDM tolerance for SCF                       =     0.001000 eV
redata: Require H convergence for SCF               =   T
redata: Hamiltonian tolerance for SCF               =     0.001000 eV
redata: Require (free) Energy convergence for SCF   =   F
redata: (free) Energy tolerance for SCF             =     0.000100 eV
redata: Using DFT-D3 dispersion                     =   F
redata: Using Saved Data (generic)                  =   F
redata: Use continuation files for DM               =   F
redata: Neglect nonoverlap interactions             =   F
redata: Method of Calculation                       = Diagonalization
redata: Electronic Temperature                      =   299.9863 K
redata: Fix the spin of the system                  =   F
redata: Max. number of TDED Iter                    =        1
redata: Number of TDED substeps                     =        3
redata: Dynamics option                             = Single-point calculation
mix.SCF: Pulay mixing                            = Pulay
mix.SCF:    Variant                              = stable
mix.SCF:    History steps                        = 2
mix.SCF:    Linear mixing weight                 =     0.250000
mix.SCF:    Mixing weight                        =     0.250000
mix.SCF:    SVD condition                        = 0.1000E-07
redata: Save all siesta data in one NC              =   F
redata: ***********************************************************************

%block SCF.Mixers
  Pulay
%endblock SCF.Mixers

%block SCF.Mixer.Pulay
  # Mixing method
  method pulay
  variant stable

  # Mixing options
  weight 0.2500
  weight.linear 0.2500
  history 2
%endblock SCF.Mixer.Pulay

DM_history_depth set to one: no extrapolation allowed by default for geometry relaxation
Size of DM history Fstack: 1
Total number of electrons:    14.691200
Total ionic charge:    14.691200

siesta: k-grid: Number of k-points = 1
siesta: k-point is Gamma-only

diag: Algorithm                                     = D&C
diag: Used triangular part                          = Lower
diag: Absolute tolerance                            =  0.100E-15
diag: Orthogonalization factor                      =  0.100E-05
diag: Memory factor                                 =  1.0000


ts: **************************************************************
ts: Save H and S matrices                           =    F
ts: Save DM and EDM matrices                        =    F
ts: Only save the overlap matrix S                  =    F
ts: **************************************************************

************************ Begin: TS CHECKS AND WARNINGS ************************
************************ End: TS CHECKS AND WARNINGS **************************


                     ====================================
                        Single-point calculation
                     ====================================

outcell: Unit cell vectors (Ang):
        3.000000    0.000000    0.000000
        0.000000   10.000000    0.000000
        0.000000    0.000000    5.000000

outcell: Cell vector modules (Ang)   :    3.000000   10.000000    5.000000
outcell: Cell angles (23,13,12) (deg):     90.0000     90.0000     90.0000
outcell: Cell volume (Ang**3)        :    150.0000
Gamma-point calculation with interaction between periodic images
Some features might not work optimally:
e.g. DM initialization from atomic data
<dSpData1D:S at geom step 0
  <sparsity:sparsity for geom step 0
    nrows_g=34 nrows=34 sparsity=.8599 nnzs=994, refcount: 7>
  <dData1D:(new from dSpData1D) n=994, refcount: 1>
refcount: 1>
new_DM -- step:     1
Initializing Density Matrix...
DM filled with atomic data:
<dSpData2D:DM initialized from atoms
  <sparsity:sparsity for geom step 0
    nrows_g=34 nrows=34 sparsity=.8599 nnzs=994, refcount: 8>
  <dData2D:DM n=994 m=1, refcount: 1>
refcount: 1>
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:       6      41

InitMesh: MESH = 32 x 108 x 54 = 186624
InitMesh: Mesh cutoff (required, used) =   300.000   314.456 Ry
New grid distribution [1]: sub = 2

stepf: Fermi-Dirac step function

siesta: Program's energy decomposition (eV):
siesta: Ebs     =      -180.021798
siesta: Eions   =      1305.402760
siesta: Ena     =       345.415034
siesta: Ekin    =       555.535407
siesta: Enl     =       -74.832087
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =       -26.513197
siesta: DUscf   =         3.820796
siesta: DUext   =         0.000000
siesta: Ex      =      -168.409517
siesta: Ec      =       -22.415369
siesta: Exc     =      -190.824886
siesta: EbV     =         0.000000
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =      -684.103699
siesta: Etot    =      -692.801693
siesta: FreeEng =      -692.867026

        iscf     Eharris(eV)        E_KS(eV)     FreeEng(eV)     dDmax    Ef(eV) dHmax(eV)
   scf:    1     -684.103699     -692.801693     -692.867026  0.600010 -9.027268  3.516886
timer: Routine,Calls,Time,% = IterSCF        1       0.055   6.81
   scf:    2     -694.790787     -693.959440     -694.024727  0.113790 -8.335249  1.319074
   scf:    3     -694.201912     -694.151344     -694.216587  0.077324 -7.963940  0.181169
   scf:    4     -694.156760     -694.154432     -694.219681  0.004864 -8.005817  0.074856
   scf:    5     -694.156044     -694.155253     -694.220504  0.001611 -8.017206  0.062662
   scf:    6     -694.154595     -694.155378     -694.220631  0.008227 -8.037400  0.144304
   scf:    7     -693.999237     -694.107814     -694.172911  0.048623 -8.025318  0.993735
   scf:    8     -694.158295     -694.156150     -694.221406  0.042352 -8.033858  0.031728
   scf:    9     -694.156174     -694.156162     -694.221418  0.000214 -8.032826  0.027512
   scf:   10     -693.810743     -694.041998     -694.106880  0.065263 -8.030889  1.511546
   scf:   11     -694.155583     -694.156186     -694.221441  0.064560 -8.032096  0.010891
   scf:   12     -694.156192     -694.156189     -694.221445  0.000134 -8.031639  0.006965
   scf:   13     -694.153569     -694.155309     -694.220562  0.005660 -8.030815  0.133017
   scf:   14     -694.156015     -694.156184     -694.221440  0.006199 -8.031335  0.014544
   scf:   15     -694.156195     -694.156190     -694.221446  0.000326 -8.031084  0.006777
   scf:   16     -694.156124     -694.156166     -694.221421  0.000816 -8.030807  0.023920
   scf:   17     -694.156118     -694.156179     -694.221435  0.001669 -8.030981  0.017317
   scf:   18     -694.156196     -694.156192     -694.221448  0.000566 -8.030806  0.003986
   scf:   19     -694.156189     -694.156191     -694.221447  0.000124 -8.030699  0.006141
   scf:   20     -694.156078     -694.156159     -694.221415  0.001343 -8.030806  0.026627
   scf:   21     -694.156195     -694.156192     -694.221448  0.001066 -8.030664  0.001699
   scf:   22     -694.156192     -694.156192     -694.221448  0.000014 -8.030618  0.001689
   scf:   23     -694.155491     -694.155962     -694.221217  0.002948 -8.030657  0.068005
   scf:   24     -694.156193     -694.156192     -694.221448  0.002893 -8.030590  0.000454
   scf:   25     -694.156192     -694.156192     -694.221448  0.000006 -8.030568  0.000378

SCF Convergence by DM+H criterion
max |DM_out - DM_in|         :     0.0000064810
max |H_out - H_in|      (eV) :     0.0003776718
SCF cycle converged after 25 iterations

Using DM_out to compute the final energy and forces
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:       6      41

siesta: E_KS(eV) =             -694.1562

siesta: E_KS - E_eggbox =      -694.1562

siesta: Atomic forces (eV/Ang):
----------------------------------------
   Tot    0.000000   -0.000000   -0.000000
----------------------------------------
   Max    5.139263
   Res    2.328131    sqrt( Sum f_i^2 / 3N )
----------------------------------------
   Max    5.139263    constrained

Stress tensor Voigt[x,y,z,yz,xz,xy] (kbar):     -210.37      -62.74       -0.74       -0.00        0.00       -0.00
(Free)E + p*V (eV/cell)     -685.6752
Target enthalpy (eV/cell)     -694.2214

siesta: Program's energy decomposition (eV):
siesta: Ebs     =      -204.385120
siesta: Eions   =      1305.402760
siesta: Ena     =       345.415034
siesta: Ekin    =       525.910805
siesta: Enl     =       -68.042694
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =        -7.502823
siesta: DUscf   =         1.317496
siesta: DUext   =         0.000000
siesta: Ex      =      -163.677443
siesta: Ec      =       -22.173808
siesta: Exc     =      -185.851251
siesta: EbV     =         0.000000
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =      -694.156192
siesta: Etot    =      -694.156192
siesta: FreeEng =      -694.221448

siesta: Final energy (eV):
siesta:  Band Struct. =    -204.385120
siesta:       Kinetic =     525.910805
siesta:       Hartree =     468.105972
siesta:         Edftu =       0.000000
siesta:       Eso     =       0.000000
siesta:    Ext. field =       0.000000
siesta:         Exch. =    -163.677443
siesta:         Corr. =     -22.173808
siesta:     Bulk bias =       0.000000
siesta:   Exch.-corr. =    -185.851251
siesta:  Ion-electron =   -1423.239813
siesta:       Ion-ion =     -79.081906
siesta:       Ekinion =       0.000000
siesta: D3 dispersion =       0.000000
siesta:         Total =    -694.156192
siesta:         Fermi =      -8.030568

siesta: Atomic forces (eV/Ang):
siesta:      1    0.000000   -5.139231   -0.000000
siesta:      2    2.116471    2.589489   -0.000000
siesta:      3   -2.116471    2.589489   -0.000000
siesta:      4    0.000000    5.139263   -0.000000
siesta:      5    2.116476   -2.589505   -0.000000
siesta:      6   -2.116476   -2.589505   -0.000000
siesta: ----------------------------------------
siesta:    Tot    0.000000   -0.000000   -0.000000

siesta: Stress tensor (static) (eV/Ang**3):
siesta:    -0.131305   -0.000000   -0.000000
siesta:    -0.000000   -0.039158    0.000000
siesta:     0.000000   -0.000000   -0.000463

siesta: Cell volume =        150.000000 Ang**3

siesta: Pressure (static):
siesta:                Solid            Molecule  Units
siesta:           0.00062054          0.00032333  Ry/Bohr**3
siesta:           0.05697517          0.02968689  eV/Ang**3
siesta:          91.28428370         47.56363544  kBar
(Free)E+ p_basis*V_orbitals  =        -692.917111
(Free)Eharris+ p_basis*V_orbitals  =        -692.917111

siesta: Electric dipole (a.u.)  =   -0.000000   -0.000056   -0.000000
siesta: Electric dipole (Debye) =   -0.000000   -0.000141   -0.000000

cite: Please see "synth_atom.bib" for an exhaustive BiBTeX file.
cite: Please clearly indicate Siesta version in published work: 5.0.0-alpha-123-gfb8fd7d02
cite: This calculation has made use of the following articles
cite: which are encouraged to be cited in a published work.
        Primary SIESTA paper
          DOI: www.doi.org/10.1088/0953-8984/14/11/302


timer: Elapsed wall time (sec) =       2.491
timer: CPU execution times (sec):

Routine            Calls   Time/call    Tot.time        %
siesta                 1       2.483       2.483   100.00
Setup                  1       0.063       0.063     2.55
bands                  1       0.000       0.000     0.00
KSV_init               1       0.000       0.000     0.00
IterGeom               1       2.419       2.419    97.43
geom_init              1       0.692       0.692    27.88
state_init             1       0.050       0.050     1.99
hsparse                1       0.000       0.000     0.02
overlap                1       0.049       0.049     1.96
Setup_H0               1       0.588       0.588    23.69
naefs                  2       0.000       0.000     0.02
MolMec                 2       0.000       0.000     0.00
kinefsm                2       0.024       0.047     1.90
nlefsm                 2       0.204       0.408    16.42
DHSCF_Init             1       0.135       0.135     5.44
DHSCF1                 1       0.010       0.010     0.42
INITMESH               1       0.000       0.000     0.00
DHSCF2                 1       0.124       0.124     5.01
REMESH                 1       0.007       0.007     0.30
REORD                 62       0.000       0.009     0.36
PHION                  1       0.096       0.096     3.86
COMM_BSC             111       0.000       0.012     0.50
POISON                28       0.004       0.126     5.07
fft                   56       0.002       0.086     3.45
setup_H               26       0.054       1.398    56.29
DHSCF                 27       0.066       1.769    71.22
DHSCF3                27       0.054       1.448    58.31
rhoofd                27       0.020       0.553    22.26
XC                    27       0.012       0.322    12.96
GXC-CellXC            27       0.011       0.308    12.42
gridxc@cellXC         27       0.011       0.308    12.41
vmat                  27       0.015       0.400    16.12
IterSCF               25       0.054       1.350    54.37
compute_dm            25       0.000       0.004     0.18
diagon                25       0.000       0.004     0.17
r-eigvec              25       0.000       0.004     0.14
r-buildHS             25       0.000       0.000     0.00
rdiag                 25       0.000       0.003     0.14
rdiag1                25       0.000       0.000     0.01
rdiag2                25       0.000       0.000     0.01
rdiag3                25       0.000       0.002     0.09
rdiag4                25       0.000       0.000     0.01
r-buildD              25       0.000       0.000     0.02
MIXER                 24       0.000       0.000     0.01
PostSCF                1       0.377       0.377    15.16
DHSCF4                 1       0.320       0.320    12.91
dfscf                  1       0.301       0.301    12.10
overfsm                1       0.001       0.001     0.03
state_analysis         1       0.000       0.000     0.01
siesta_move            1       0.000       0.000     0.00
Analysis               1       0.000       0.000     0.01
optical                1       0.000       0.000     0.00
  

>> End of run:  28-SEP-2023  15:40:34
Job completed
